tests/test_language_java.py

```diff
@@ -64,3 +70,18 @@ def test_latest_maven_version():
         assert "3.3.0" == java.latest_maven_version(
             group_id="com.google.cloud", artifact_id="libraries-bom"
         )
+
+
+def test_working_common_templates():
+    with tempfile.TemporaryDirectory() as tempdir:
+        workdir = shutil.copytree(
+            FIXTURES / "java_templates" / "standard", Path(tempdir) / "standard"
+        )
+        cwd = os.getcwd()
+        os.chdir(workdir)
+
+        # generate the common templates
+        java.common_templates()
+        assert os.path.isfile("README.md")
```

Comment:
If the assertion fails, then line 87 will never execute and change the directory back.  That may cause subsequent tests to fail.  Add a `try:` on line 82 and a `finally:` on line 86.

Reply:
Done. At some point it might be nice to have a context helper for changing the working directory.